<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="SmokeSuite"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="1806.130">
  <properties />
            <testcase name="TestProductSearchByUniID" time="1806.130" classname="icix.Tests.TC10782_Test">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:513)
  org.testng.Assert.assertTrue(Assert.java:42)
  org.testng.Assert.assertTrue(Assert.java:52)
  icix.Modules.Product.ProductSearchByUniID(Product.java:233)
  icix.Tests.TC10782_Test.TestProductSearchByUniID(TC10782_Test.java:27)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:601)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
